### ISSUE TITLE
BGRA format from the camera

### ISSUE THREAD

**Issue by shersoni610** [opencv/opencv#23543]
### Descripe the feature and motivation

Hello,  

OpenCV gives BGR image from the camera after converting BGRA to BGR. Is it possible to get BGRA images from a camera?

Thanks

**Comment by mediba123** [comment]
Yes it possible OpenCV gives the VideoCapture class which can be used to capture frames from a camera or a video file. By default, it captures frames in the BGR format. you can set the capture property cv2.CAP_PROP_CONVERT_RGB to False to get images in the native format of the camera.
